packs/github/config.schema.yaml

```diff
@@ -0,0 +1,54 @@
+---
+token:
+  description: "GitHub oAuth Token"
+  type: "string"
+  default: "{{system.github_oauth_token}}"
+  required: true
+
+user: 
+  description: "GitHub Username"
+  type: "string"
+
+password:
+  description: "GitHub Password"
+  type: "string"
+
+github_type:
+  description: "Default to either github or enterprise."
+  type: "string"
+  default: "online"
```

Comment:
Maybe `public` or similar would be better than `online` but not a big deal anyway.